src/components/TasksListItem/TasksListItem.tsx

```diff
@@ -68,19 +75,22 @@ export const TasksListItem = ({
           <div className='Layer__tasks-list-item__body-info'>
             <Text size={TextSize.sm}>{task.question}</Text>
             <Textarea
-              disabled={task.status === 'COMPLETED'}
-              placeholder={userResponse}
+              value={userResponse}
               onChange={(e: React.ChangeEvent<HTMLTextAreaElement>) =>
                 setUserResponse(e.target.value)
               }
             />
             <div className='Layer__tasks-list-item__actions'>
               <Button
                 disabled={
-                  task.status === 'COMPLETED' || userResponse.length === 0
+                  userResponse.length === 0 ||
+                  userResponse === task.user_response
                 }
                 variant={ButtonVariant.secondary}
-                onClick={() => submitResponseToTask(task.id, userResponse)}
+                onClick={() => {
+                  submitResponseToTask(task.id, userResponse)
+                  setIsOpen(false)
```

Comment:
After submitting, I want the task to close and go out of the way rather than stay open, which made it seem to me like I needed to do something else still.